config/config.go

```diff
@@ -4,11 +4,21 @@ import (
 	"math"
 	"time"
 
-	"github.com/Trendyol/go-dcp/helpers"
-
 	"github.com/Trendyol/go-dcp/config"
+	"github.com/Trendyol/go-dcp/helpers"
+	"github.com/segmentio/kafka-go"
 )
 
+var balancerMap = map[string]kafka.Balancer{
```

Comment:
Can we write these balancers strings to README.md as well?

Reply:
Yup